File: Step4/Count-Digits/Solution.java
class Main {
    static int countDigit(int n){
    int num = n;
    int count = 0;
    while(num>0){
        ++count;
        num = num/10;
    }
    return count;
    }
    public static void main(String[] args) {
        int result = countDigit(893);
        System.out.print(result);
    }
}

File: Step4/Count-Digits/Solution.cpp
#include <iostream>
using namespace std;
int countDigit(int n){
    int num = n;
    int count = 0;
    while(num%10 != 0){
        num = num/10;
        ++count;
    }
    return count;
}
int main() {
    int output = countDigit(5289);
    cout<<output;
    return 0;
}
